google/cloud/aiplatform/datasets/tabular_dataset.py

```diff
@@ -39,6 +39,29 @@ class TabularDataset(datasets._Dataset):
         schema.dataset.metadata.tabular,
     )
 
+    def auto_column_specs(self, target_column: str) -> Dict[str, str]:
```

Comment:
Start function with a verb?

Reply:
We decided offline to move to the training job and just use the target_column property.